daemon/api.go

```diff
@@ -279,6 +279,16 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 		return InternalError("cannot get user auth data: %s", err)
 	}
 
+	refreshInfo := client.RefreshInfo{
+		Last: formatRefreshTime(lastRefresh),
+		Next: formatRefreshTime(nextRefresh),
+	}
+	if !legacySchedule {
+		refreshInfo.Timer = refreshScheduleStr
```

Comment:
This looks good. Seems to be the field documentation that is wrong. I suggest just dropping the "The field is omitted" part of both comments. The desired intention there seems to be implied by omitempty already.